eslintrc.json

```diff
@@ -1,169 +1,166 @@
-    "no-tabs": 2,
-    "no-trailing-spaces": 2,
-    "no-underscore-dangle": [2, { "allow": ["_id"] } ],
-    "no-unneeded-ternary": 2,
-    "no-whitespace-before-property": 2,
-    "nonblock-statement-body-position": 2,
-    "object-curly-spacing": [2, "always"],
-    "object-property-newline": [2, { "allowMultiplePropertiesPerLine": true }],
-    "one-var-declaration-per-line": 2,
-    "operator-assignment": 2,
-    "operator-linebreak": [2, "before"],
-    "padded-blocks": [2, "never"],
-    "quote-props": [2, "as-needed"],
-    "quotes": [2, "single", "avoid-escape"],
-    "semi": [2, "never"],
-    "semi-spacing": 2,
-    "space-before-blocks": 2,
-    "space-before-function-paren": [2, "never"],
-    "space-in-parens": 2,
-    "space-infix-ops": 2,
-    "space-unary-ops": 2,
-    "spaced-comment": 2,
-    "template-tag-spacing": [2, "always"],
-    "arrow-spacing": 2,
-    "generator-star-spacing": 2,
+    "no-inline-comments": "error",
+    "no-lonely-if": "error",
+    "no-mixed-operators": "error",
+    "no-multi-assign": "error",
+    "no-multiple-empty-lines": "error",
+    "no-nested-ternary": "error",
+    "no-new-object": "error",
+    "no-plusplus": ["error", { "allowForLoopAfterthoughts": true }],
+    "no-tabs": "error",
+    "no-trailing-spaces": "error",
+    "no-underscore-dangle": ["error", { "allow": ["_id"] } ],
+    "no-unneeded-ternary": "error",
+    "no-whitespace-before-property": "error",
+    "nonblock-statement-body-position": "error",
+    "object-curly-spacing": ["error", "always"],
+    "object-property-newline": ["error", { "allowMultiplePropertiesPerLine": true }],
+    "one-var-declaration-per-line": "error",
+    "operator-assignment": "error",
+    "operator-linebreak": ["error", "before"],
+    "padded-blocks": ["error", "never"],
+    "quote-props": ["error", "as-needed"],
+    "quotes": ["error", "single", "avoid-escape"],
+    "semi": ["error", "never"],
+    "semi-spacing": "error",
+    "space-before-blocks": "error",
+    "space-before-function-paren": ["error", "never"],
+    "space-in-parens": "error",
+    "space-infix-ops": "error",
+    "space-unary-ops": "error",
+    "spaced-comment": "error",
+    "template-tag-spacing": ["error", "always"],
+    "arrow-spacing": "error",
+    "generator-star-spacing": "error",
     "no-confusing-arrow": ["error", {"allowParens": true}],
-    "no-duplicate-imports": 2,
-    "no-useless-computed-key": 2,
-    "no-useless-constructor": 2,
-    "no-useless-rename": 2,
-    "no-var": 2,
-    "object-shorthand": 2,
-    "prefer-arrow-callback": [2, { "allowNamedFunctions": true }],
-    "prefer-const": 2,
-    "prefer-destructuring": 2,
-    "prefer-rest-params": 2,
-    "prefer-spread": 2,
-    "prefer-template": 2,
-    "rest-spread-spacing": 2,
-    "sort-imports": 2,
-    "symbol-description": 2,
-    "template-curly-spacing": 2,
-    "yield-star-spacing": 2,
-    "curly": 2,
-    "eqeqeq": 2
-  },
-  "parserOptions": {
```

Comment:
Why is this removed?

Reply:
The `parserOptions`, it is been moved up in the file before the rules for better surfacing it and not hiding in the bottom of the file after the bigger rules block. If your comment is about the `ecmaFeature` has been conflated in the `ecmaVersion: 2018` by eslint.